course/include/isometry.h

```diff
@@ -141,7 +143,57 @@ class Matrix3{
     private:
         // Ordered by rows.
         Vector3 r1_, r2_, r3_;
-};
+
+};  // Matrix3
+
+class Isometry {
+
+    public:
+        // Constructors
+        Isometry(const Vector3& translation_vector = Vector3(), 
+                 const Matrix3& rotation_matrix = Matrix3::kIdentity) :
+                translation_vector_{translation_vector},
+                rotation_matrix_{rotation_matrix} {}
+        Isometry(const Isometry& t) : 
+            translation_vector_{t.translation_vector_},
+            rotation_matrix_{t.rotation_matrix_} {}
+
+        // Static operations
+        static Isometry FromTranslation(const Vector3&);
+        static Isometry RotateAround(const Vector3&, const double);
+        static Isometry FromEulerAngles(const double, const double, const double);
```

Comment:
It's usually a good idea to leave the argument names, so that the reader knows what each argument is.

Reply:
Addressed. I agree!